test/1_test_fullpath.js

```diff
@@ -1536,13 +1522,13 @@ contract('Voucher tests - UNHAPPY PATH', async (addresses) => {
 
       await contractVoucherKernel.triggerExpiration(tokenVoucherKey1);
 
-      let statusAfter = await contractVoucherKernel.vouchersStatus(
+      let statusAfter = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey1
       );
 
       // [1001.0000] = hex"90" = 144 = EXPIRED
       assert.equal(
-        web3.utils.toHex(statusAfter.status),
+        web3.utils.toHex(statusAfter[0]),
```

Comment:
Please use constants instead of hard-coded indexes for statusAfter (same as remark above)

Reply:
Remaining hard-coded indexes at lines 1602 + 1630 + 1663